py/desispec/scripts/exposuretable.py

```diff
@@ -0,0 +1,105 @@
+
+import os
+import numpy as np
+from astropy.table import Table
+from astropy.io import fits
+## Import some helper functions, you can see their definitions by uncomenting the bash shell command
+
+from desispec.workflow.exptable import summarize_exposure, default_exptypes_for_exptable, get_survey_definitions,\
+                                       instantiate_exposure_table, get_exposure_table_name, night_to_month, \
+                                       get_surveynum, get_exposure_table_path, get_exposure_table_name, night_to_month, \
+                                       get_exposure_table_column_defs
+from desispec.workflow.os import define_variable_from_environment, listpath, pathjoin
+from desispec.workflow.timing import get_night_banner
+from desispec.workflow.tableio import write_table
+
+
+
+def create_exposure_tables(nights, path_to_data=None, exp_table_path=None, science_types=None, \
+                           exp_filetype='csv', verbose=False, overwrite_files=False):
```

Comment:
As part of documenting this, we'll need documentation about what all of the columns in the exposure table are.  I think this is the table that a human would edit to override things like marking a specific exposure as bad (or even fixing an incorrect keyword?), so that procedure will need to be clearly documented too.

Reply:
I fully agree. In exptable there is a function that gives the names, types, and defaults. This is my way of documenting temporarily while I get feedback. Once it solidifies, then a data model should probably be made. The same is true with proctable.py.

Reply:
I've added a rudimentary explanation of each table's columns in the exptable.py and proctable.py files. Hopefully, these can serve as pseudo datamodels until the things are ironed out enough to justify creating more permanent documentation.